ptpython/repl.py

```diff
@@ -323,12 +323,14 @@ def _format_result_output(self, result: object) -> StyleAndTextTuples:
             if self.enable_output_formatting:
                 # Inline import. Slightly speed up start-up time if black is
                 # not used.
-                import black
-
-                result_repr = black.format_str(
-                    result_repr,
-                    mode=black.FileMode(line_length=self.app.output.get_size().columns),
-                )
+                try:
+                    import black
```

Comment:
Would it be possible to have only the ``import`` line in the ``try`` block and everything else in the ``else`` block?

```python
try:
    import black
except ImportError:
    pass  # Black not found
else:
    result_repr = black.format_str(...)
```
  
      
        
        
  

        <div data-src-base="/prompt-toolkit/ptpython/pull/458/review_comment/suggestion_button" class="js-suggestion-button-placeholder"><div class="d-flex d-md-inline-block mr-0 mr-md-3">
  
      
    


</div>
</div><!--EndFragment-->

Reply:
sure, I didn't known at all 'try except else ' was a possible consturction, 5'